### DISCUSSION TITLE
One user with EP, the other not

### DISCUSSION THREAD

**Discussion by cafuar** [valinet/ExplorerPatcher#2784] [General]
I have a PC with 2 users, one uses EP and the other does not. If both users are logged in and switch from one to the other, the screen goes black and only the cursor is visible. I can't do anything, I have to restart the PC, no key combination works, could it be a problem with EP?
Thank you.

**Comment by pyrates999** [comment]
EP is enabled for all users when it is installed.

Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.  These should be screen shots.

Please include any other software you have installed that modifies the UI of windows.

**Reply by cafuar** [reply]
The second user was created some time after installing EP on the main user. The second user appears looking like Windows 11, EP is not active on that user.
Winver: 23H2 22631.3085
EP: 22621.2861.62.2

**Comment by pyrates999** [comment]
EP is always active since it is system wide.

**Comment by pyrates999** [comment]
You also have installed a preview update: https://support.microsoft.com/en-au/topic/january-23-2024-kb5034204-os-builds-22621-3085-and-22631-3085-preview-7652acf2-56dc-430e-b8ef-ec8f56ec1028

I would first try to uninstall that preview update.

In the settings app, untick "Get the latest updates as soon as available" so that preview updates are not automatically installed.

All preview updates are then shown as download and install and are never automatically installed if you have the above unticked.

Preview updates are also not fully tested by Microsoft and thus could have bugs.

You should only be installing preview updates if you are a developer or are OK with running pre-release versions of EP.

**Comment by cafuar** [comment]
Ok, thanks for your comment, I'll try it
